funds-manager/funds-manager-server/src/execution_client/swap.rs

```diff
@@ -123,7 +128,8 @@ impl ExecutionClient {
         }
 
         // Otherwise, approve the allowance
-        let tx = erc20.approve(spender, amount);
+        let approval_amount = amount * U256::from(APPROVAL_AMPLIFIER);
```

Comment:
nit: we can make `APPROVAL_AMPLIFIER` a constant `U256` by using `U256::from_limbs`, which is a `const fn`